libimage/image.go

```diff
@@ -775,6 +775,7 @@ func (i *Image) Unmount(force bool) error {
 
 // Size computes the size of the image layers and associated data.
 func (i *Image) Size() (int64, error) {
+	// TODO: cache the result to optimize performance of subsequent calls
```

Comment:
Implementation cache for this rather looks complex at first glance, if i gauge this right.

* Store `image` size in boltdb.
* Ensure to update the boltdb on every `pull` or `rmi`
* Ensure function is locked and waits for any parallel `pull` or `rmi` to complete.

Reply:
Each `Image` object has an internal cache (see https://github.com/vrothberg/common/blob/baebe0edc671bd3f2e8e18513551df126d025186/libimage/image.go#L38).  That's where I want to store it.